packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library/components/Icon"
+import classnames from "classnames"
+import {
+  FieldGroup,
+  FieldMessage,
+  Input,
+  InputProps,
+  Label,
+} from "@kaizen/draft-form/KaizenDraft/Form"
+import { renderValidationMessage } from "../../../utils/renderValidationMessage"
+
+import styles from "./DateInput.scss"
+
+type OmittedInputProps =
+  | "startIconAdornment"
+  | "endIconAdornment"
+  | "inputType"
+  | "inputValue"
+
+export interface validationMessagesProps {
+  error?: string | React.ReactNode
+}
+
+export interface DateInputProps extends Omit<InputProps, OmittedInputProps> {
+  id: string
+  calendarId?: string
+  buttonRef?: React.RefObject<HTMLButtonElement>
+  labelText: string | React.ReactNode
+  inline?: boolean
+  icon: React.SVGAttributes<SVGSymbolElement>
+  /**
+   * A description that provides context for the text field
+   */
+  description?: string | React.ReactNode
+  isOpen: boolean
+  onButtonClick: () => void
+  onKeyDown: (e: React.KeyboardEvent<HTMLDivElement>) => void
+  handleOnBlur?: React.FocusEventHandler<HTMLInputElement> | undefined
+}
+
+export const DateInput: React.VFC<DateInputProps> = ({
+  id,
+  disabled = false,
+  buttonRef,
+  labelText,
+  defaultInputValue,
+  inputRef,
+  value,
+  description,
+  reversed = false,
+  inline = false,
+  status = "default",
+  icon,
+  onButtonClick,
+  calendarId,
+  isOpen,
+  onKeyDown,
+  handleOnBlur,
+  ...inputProps
+}) => (
+  <FieldGroup
+    id={`${id}-field-group`}
+    automationId={`${id}-field-group`}
+    inline={inline}
```

Comment:
You can just hardcode this to `true` and remove the `inline` prop.

This prop was used to remove built in margins, which we've moved away from.